src/main/java/utilties/models/Game.java

```diff
@@ -14,6 +16,9 @@
     private Player player;
     private Player opponent;
 
+    private ArrayList<PlayerScriptCommand> playerCommands;
+    private ArrayList<PlayerScriptCommand> opponentCommands;
+
```

Comment:
If we opt to go with passing separate parameter lists in, we could add two more ArrayLists to store the parameters for each _performFunction_ command.  Then when we utilize our Drag and Drop interface, just add them when you determine the command to use.

`private ArrayList<Parameter> playerCommandParameters;
private ArrayList<Parameter> opponentCommandParameters;
`